### DISCUSSION TITLE
如何获取自化？

### DISCUSSION THREAD

**Discussion by l2x** [SylarLong/iztro#214] [Q&A]
你好，如何获取自化？
FunctionalPalace selfMutaged、selfMutagedOneOf 返回宫位是否有自化
如何判断宫位中的某星产生了哪种自化？
谢谢。

![image](https://github.com/user-attachments/assets/e54af485-63c1-40d6-a294-fa37701f0132)

**Comment by SylarLong** [comment]
请参考：https://github.com/SylarLong/react-iztro/blob/main/src/Izstar/Izstar.tsx#L45-L58
